src/test/java/nextstep/subway/line/LineSteps.java

```diff
@@ -43,11 +51,11 @@
                 .then().log().all().extract();
     }
 
-    public static ExtractableResponse<Response> 지하철_노선_제거_요청() {
+    public static ExtractableResponse<Response> 지하철_노선_제거_요청(Long lineId) {
         return RestAssured
                 .given().log().all()
                 .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .when().delete("/lines/1")
+                .when().delete("/lines/" + lineId)
```

Comment:
가변인자로 pathParameter를 넘겨서 사용할 수도 있어요 :)
```suggestion
                .when().delete("/lines/{lineId}", lineId)
```